### ISSUE TITLE
Distorted model and wrong reconstruction

### ISSUE THREAD

**Issue by Feriel-HA** [alicevision/Meshroom#1791]
Hy everyone. 
I am a begginer in meshroom and photogrammetry... I tried to reconstruct a building but the model resulting was completely distorted, and cameras reconstruction and positionning was wrong. 
I used nikon D3200, sift+ akaze description types, sequential and vocabulary tree method, cross and guided matching, match from known camera poses, and min input track length 6.
I want to know if the problem came from my dataset ?... the background in some pictures ? my nodes settings ?? I need some help please
Here some of the pictures of the building
![CP](https://user-images.githubusercontent.com/115005822/193824041-befc1d1a-06a6-4a1f-8e73-7b2e6be36157.jpg)
![AKK](https://user-images.githubusercontent.com/115005822/193824093-d68e5fe8-a08c-474c-8cbe-4be6723b73d4.JPG)
![CAA](https://user-images.githubusercontent.com/115005822/193824268-3ff955fe-6ef1-46c5-a5bf-cdfb263331ad.jpg)
![AZ](https://user-images.githubusercontent.com/115005822/193824311-1c622d46-933f-4d08-ade8-36bd8a25ea82.jpg)
And here thé resulting model
![Capture2](https://user-images.githubusercontent.com/115005822/193825743-0f5e0cfd-f6c5-4108-9143-cf2f5fdc58da.PNG)
![Capture](https://user-images.githubusercontent.com/115005822/193825779-1866efc5-d45b-4382-9c85-30d4229be08c.PNG)

**Comment by msanta** [comment]
It is hard to say without seeing the dataset you used. Are you able to share it? Are the photos you showed part of your dataset? I'm asking because the lighting conditions differ in the photos (sunny and overcast).

**Comment by Feriel-HA** [comment]
> It is hard to say without seeing the dataset you used. Are you able to share it? Are the photos you showed part of your dataset? I'm asking because the lighting conditions differ in the photos (sunny and overcast).

**Comment by Feriel-HA** [comment]
> It is hard to say without seeing the dataset you used. Are you able to share it? Are the photos you showed part of your dataset? I'm asking because the lighting conditions differ in the photos (sunny and overcast).

**Comment by Feriel-HA** [comment]
>

**Comment by Feriel-HA** [comment]
Yes those photos come from my dataset. Actually the weather was overcast the day I took the pictures... Sometimes sunny and sometimes cloudy. 
You can find some of the dataset images here:

https://drive.google.com/drive/folders/1wCLlSMIN_w6w2izjMprNo0n8tTZ3z6OA

I red that to make a convinient reconstruction of à building we must have an important number of photos, and to cast shadows too....

**Comment by msanta** [comment]
From the thumbnails it looks like you are not getting enough photos from different angles. Try getting a good 360 degree coverage. 

In general if you are taking photos outside it should be on an overcast day so that changes in lighting condition don't cause issues. Also make sure that photos are not over/under exposed.

**Comment by natowi** [comment]
...also start with the default parameters and change them if the results are not as expected. In most cases the defaults are a good start. See https://github.com/alicevision/Meshroom/wiki/Reconstruction-parameters for details.
